electrode_models/air_electrode.py

```diff
@@ -55,6 +55,14 @@ def __init__(self, input_file, inputs, sep_inputs, counter_inputs,
 
         # Phase volume fractions
         eps_host = np.array([inputs['eps_host']])
+        eps_host = eps_host[0]
+
+        # current = params['i_ext']
+        # # Split the current from the units:
+        # i_ext, units = current.split()
+        self.surf_to_volume = 1.23224E-11
```

Comment:
a. Is this setable from the input file?

b. This number seems like it is inverted.  A/V should be proportional to 1/r, no?

Reply:
I'm not using this for anything, so I'll remove this line